src/Docker.PowerShell/Cmdlets/DockerFactory.cs

```diff
@@ -13,19 +14,17 @@ public class DockerFactory
     public static DockerClient CreateClient(string HostAddress, string CertificateLocation)
     {
         HostAddress = HostAddress ?? Environment.GetEnvironmentVariable("DOCKER_HOST") ?? "npipe://./pipe/docker_engine";
-        CertificateLocation = CertificateLocation ?? Environment.GetEnvironmentVariable("DOCKER_CERT_PATH");
-
+        
         CertificateCredentials cred = null;
-        if (!string.IsNullOrEmpty(CertificateLocation))
+        if (!string.IsNullOrEmpty(CertificateLocation) || Environment.GetEnvironmentVariable("DOCKER_TLS_VERIFY") == "1")
         {
             // Try to find a certificate for secure connections.
             cred = new CertificateCredentials(
                     new X509Certificate2(
-                        System.IO.Path.Combine(CertificateLocation, KeyFileName),
+                        System.IO.Path.Combine(CertificateLocation ?? Environment.GetEnvironmentVariable("DOCKER_CERT_PATH"), KeyFileName),
```

Comment:
Is there validation somewhere that if DOCKER_TLS_VERIFY == 1 than DOCKER_CERT_PATH will for sure be set? Seems like we need a prettier error here than "null ref exception"